File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/EngineDataPoint.java
package com.kendoui.spring.models;

public class EngineDataPoint {
    private int rpm;
    private double torque;
    private double power;
    
    public EngineDataPoint(int rpm, double torque, double power) {
        setRPM(rpm);
        setTorque(torque);
        setPower(power);
    }
    
    public int getRPM() {
        return rpm;
    }

    public void setRPM(int rpm) {
        this.rpm = rpm;
    }
    
    public double getTorque() {
        return torque;
    }

    public void setTorque(double torque) {
        this.torque = torque;
    }
    
    public double getPower() {
        return power;
    }

    public void setPower(double power) {
        this.power = power;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/EngineDataPoint.cs
namespace Kendo.Mvc.Examples.Models
{
    public class EngineDataPoint
    {
        public EngineDataPoint()
        {
        }

        public EngineDataPoint(int rpm, double torque, double power)
        {
            RPM = rpm;
            Torque = torque;
            Power = power;
        }

        public int RPM;
        public double Torque;
        public double Power;
    }
}
